### PR TITLE
931 limiting interviews changes

### LINKED ISSUE #931: limiting interviews changes
- [ ] Show number of interviews given to an applicant in table
- [ ] Show an applicants prioritization of the job in table
- [ ] Overview for mg_nestleder of all with less than 3 interview given, but 3 or more jobs applied to.

### PR DESCRIPTION
**Pull request by snorrekr**

### CODE REVIEW

File: app/views/admissions_admin/admissions/show.html.haml [hunk 1]
```diff
@@ -16,6 +16,11 @@
     .samf-button.blue.small.mb-2
       = t("admissions_admin.show_unflagged_applicants")+"..."
 
+  - if can? :manage, Admission and not @open_admissions.empty?
+    = link_to admissions_admin_admission_show_applicants_missing_interviews_path(@admission) do
+      .samf-button.blue.small.mb-2
+        = "Vis søkere med 3 eller flere søknader, med færre enn 3 intervjuer..."
```
Comment (snorrekr) [inline app/views/admissions_admin/admissions/show.html.haml]: I think we should change the layout of these buttons, to make it more userfriendly for mg-nestleder

File: app/views/admissions_admin/jobs/_jobs_show.html.haml [hunk 2]
```diff
@@ -54,6 +54,10 @@
       = t('interviews.interview_priority')
     %th.status
       = "Status"
+    %th.something
+      = "Rangering"
```
Comment (snorrekr) [inline app/views/admissions_admin/jobs/_jobs_show.html.haml]: Prioritering er kanskje bedre

File: app/views/admissions_admin/admissions/show.html.haml (hunk 1 above)
Comment (Frenje123) [reply app/views/admissions_admin/admissions/show.html.haml]: Agreed!

File: app/views/admissions_admin/jobs/_jobs_show.html.haml (hunk 2 above)
Comment (Frenje123) [reply app/views/admissions_admin/jobs/_jobs_show.html.haml]: There is already a field named "Priority", perhaps it is a bit misleading?

File: app/views/admissions_admin/admissions/show.html.haml [hunk 3]
```diff
@@ -13,13 +13,18 @@
...
+  - if can? :manage, Admission and not @open_admissions.empty?
+    = link_to admissions_admin_admission_show_applicants_missing_interviews_path(@admission) do
+      .samf-button.red.small.mb-2
+        = t("admissions_admin.show_less_three_interviews")
+
   - if can? :manage, Campus and not @open_admissions.empty?
     -#campus-system should be worked on, its major fucksy wucksie
     = link_to admin_admissions_admin_campus_index_path do
-      .samf-button.green
+      .samf-button.orange
```
Comment (snorrekr) [inline app/views/admissions_admin/admissions/show.html.haml]: `.samf-button.orange.small.mb-2`

**Review by Frenje123 — APPROVED** [review]

**Pull request merged**